Topic-4/src/test/java/bootcamp/exercices/ShoppingCartTest.java

```diff
@@ -0,0 +1,22 @@
+package bootcamp.exercices;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+
+public class ShoppingCartTest {
+
+	@Test
+	public void testTotal() {
+		AbstractProduct product1= new Product("0001", "notebook", 1000);
+		AbstractProduct product2= new Product("0002", "phone", 1000);
+		AbstractProduct product3= new Product("0003", "tv", 1000);
+		ShoppingCart shoppingCart= new ShoppingCart();
+		shoppingCart.addProduct(product1);
+		shoppingCart.addProduct(product2);
+		shoppingCart.addProduct(product3);
+		Assert.assertEquals(shoppingCart.getTotal(), 3000.0);
+	}
+
```

Comment:
[High] Remember that TDD approach idea is to cover all possible scenarios in your test. You should add more testing here.